packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -305,9 +305,10 @@ async function handler(req: CustomRequest) {
   const eventTrigger: WebhookTriggerEvents = "BOOKING_CANCELLED";
   // Send Webhook call if hooked to BOOKING.CANCELLED
   const subscriberOptions = {
-    userId: bookingToDelete.userId,
-    eventTypeId: (bookingToDelete.eventTypeId as number) || 0,
+    userId: !bookingToDelete.eventType?.teamId ? bookingToDelete.userId : 0,
+    eventTypeId: (bookingToDelete.eventTypeId as number) ?? 0,
     triggerEvent: eventTrigger,
+    teamId: bookingToDelete.eventType?.teamId ?? 0,
```

Comment:
nit: Instead of making these changes everywhere, I think we can handle it in getWebhooks.
if teamId is there, ignore userId(by setting it to 0)
After that it's just about reading and passing values. Zeroing them would happen in getWebhooks
```suggestion
    userId: bookingToDelete.userId,
    eventTypeId: (bookingToDelete.eventTypeId as number) ?? 0,
    triggerEvent: eventTrigger,
    teamId: bookingToDelete.eventType?.teamId,
```